apps/coscrad-frontend/src/components/dynamic-forms/dynamic-form-elements/multilingual-text-input/actions/multilingual-text-form-reducer.ts

```diff
@@ -28,6 +28,7 @@ export const multilingualTextFormReducer = (
                 items: state.items.concat(
                     state.items.concat({
                         languageCode: languageCode,
+                        defaultLanguageCode: languageCode,
```

Comment:
Reset these changes. This only affects command forms and has nothing to do with this PR.